ganga/GangaCore/GPIDev/Lib/Job/Job.py

```diff
@@ -2567,10 +2558,6 @@ def __setattr__(self, attr, value):
 
             if value != []:
 
-                if getConfig('Output')['ForbidLegacyOutput']:
```

Comment:
I would assume here we want to keep the error message by default. I think that `ForbidLegacyOutput` is by default `True` (at least going by the change you make by getting rid of the function above) - in which case we should keep reporting an error if a user tries to use the legacy output.